.github/workflows/lint_docs.yml

```diff
@@ -29,4 +29,4 @@ jobs:
       - name: Install dependencies
         run: uv sync --frozen --only-group lint
       - name: Lint docs
-        run: tools/lint_docs.sh
```

Comment:
`tools/lint_docs.sh` runs both `scan` and `fix` -- but during a CI run, changing files to fix lint errors is not needed